5-appinfra/modules/cicd-pipeline/locals.tf

```diff
@@ -14,8 +14,7 @@
 
 locals {
   cache_filename     = "cache"
-  service_name       = reverse(split("-", var.service))[0]
-  team_name          = split("-", var.service)[0]
-  service_clean      = replace(var.service, "/", "-")
+  full_service       = "${var.team_name}-${var.service_name}"
```

Comment:
in the case of frontend, would the trigger be named `frontend-frontend`?

Reply:
yeah, gonna add a check on it